From 30bda5feca562b08250bf40dd6c3c078bc3c5f8b Mon Sep 17 00:00:00 2001
From: Stephen <stephen@vamrs.com>
Date: Thu, 16 Apr 2020 18:30:13 +0800
Subject: [PATCH 93/97] net: wireless: rockchip_wlan: rtl8723du: change the
 value of TopDIR

Fix the issue of failing to build kernel out of tree.

Signed-off-by: Stephen <stephen@vamrs.com>
---
 drivers/net/wireless/rockchip_wlan/rtl8723du/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rtl8723du/Makefile b/drivers/net/wireless/rockchip_wlan/rtl8723du/Makefile
index 25b276dec975..868df6d2ba18 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8723du/Makefile
+++ b/drivers/net/wireless/rockchip_wlan/rtl8723du/Makefile
@@ -154,7 +154,7 @@ CONFIG_PLATFORM_RTL8197D = n
 
 CONFIG_DRVEXT_MODULE = n
 
-export TopDIR ?= $(shell pwd)/drivers/net/wireless/rockchip_wlan/rtl8723du
+export TopDIR ?= $(src)
 
 ########### COMMON  #################################
 ifeq ($(CONFIG_GSPI_HCI), y)
-- 
2.25.1

